### PR TITLE
deps: bump mocha-3 to mocha-4

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

Resolves https://github.com/celestiaorg/docs/issues/1018
- [x] Add new celestia-node version
- [x] adds v0.11.0-rc12 API doc (but api was not updated, so it is the same as rc11)

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: docs/nodes/mocha-testnet.mdx [hunk 1]
```diff
@@ -103,16 +95,35 @@ port is 9090. In the Cosmos SDK, gRPC is used to define state queries and
...
-* [https://grpc-celestia-testnet.mms.team:12090](https://grpc-celestia-testnet.mms.team)
-* [https://celestia-testnet.nodejumper.io:9090](https://celestia-testnet.nodejumper.io:9090)
+* `grpc-mocha-4.consensus.celestia-mocha.com:443`
+* `grpc-2-mocha-4.consensus.celestia-mocha.com:443`
+* `full.consensus.mocha-4.celestia-mocha.com:9090`
+* `consensus-full-mocha-4.celestia-mocha.com:9090`
+
+## Bridge and full node endpoints
+
+The endpoints below are for bridge and full nodes only. They can be used to
```
Comment (jcstein) [inline docs/nodes/mocha-testnet.mdx:105]: is this accurate @Bidon15 ?

File: docs/constants/mocha_versions.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const mochaVersions = Object.freeze({
-  "app-latest-tag": "v1.0.0-rc10",
-  "app-latest-sha": "96b86ba8c0a11b1b854da1c256c5d737dbaeede3",
+  "app-latest-tag": "v1.0.0-rc14",
+  "app-latest-sha": "cdc6bff4cde817a0f9fd6b30b57a02ad44698d17",
   "core-latest-tag": "v1.24.0-tm-v0.34.28",
   "core-latest-sha": "d24c81afe0ecc4ccfbb7eb543a45a11794f04db4",
   "node-latest-tag": "v0.11.0-rc8",
```
Comment (Bidon15) [inline docs/constants/mocha_versions.js:6]: we need to change this to 
```suggestion
  "node-latest-tag": "v0.11.0-rc12",
```

File: docs/nodes/mocha-testnet.mdx [hunk 3]
```diff
@@ -103,16 +95,35 @@ port is 9090. In the Cosmos SDK, gRPC is used to define state queries and
...
+
+Bridge node 1:
+
+* da-bridge-mocha-4.celestia-mocha.com
+* bridge-mocha-4.da.celestia-mocha.com
+
+Bridge node 2:
+
+* da-bridge-mocha-4-2.celestia-mocha.com
+* bridge-mocha-4.celestia-mocha.com
```
Comment (Bidon15) [inline docs/nodes/mocha-testnet.mdx:116]:
```suggestion
* bridge-mocha-4-2.da.celestia-mocha.com
```

**Review by Bidon15 — COMMENTED** [review]
Thanks for the docs bump ❤️  You 🪨 
2 small nits and we are ready to go 🚀

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**